src/main/java/utils/SeleniumDriver.java

```diff
@@ -24,11 +25,37 @@
     public final static int TIMEOUT = 50;
     public final static int PAGE_LOAD_TIMEOUT = 50;
     public static URL url;
+    public static String chromeLinux=System.getProperty("user.dir")+ ("//executables/linux/chromedriver");
+    public static String chromeWindow=System.getProperty("user.dir")+ ("//executables/window/chromedriver.exe");
+    public static String chromeMac=System.getProperty("user.dir")+ ("//executables/mac/chromedriver");
+    public static String currentOS;
 
       SeleniumDriver() {
-        System.setProperty("webdriver.chrome.driver", "./src/test/resources/webdriverFiles/chromedriver");
-        // driver = new ChromeDriver();
-        chromeDriver=new ChromeDriver();
+          currentOS=System.getProperty("os.name");
+          System.out.println("++++++++"+currentOS);
+          if(currentOS.toLowerCase().contains("mac")) {
+              System.setProperty("webdriver.chrome.driver", chromeMac);
```

Comment:
Please check if property is already set. In this case don't overwrite it. 
BTW: Where are this executables come from? What is their license? Why are we including then as binaries?